cdc-sandbox/db/shrink.sql

```diff
@@ -0,0 +1,845 @@
+	jurisdiction_derivation_ind char(1) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	provider_uid bigint NULL,
+	CONSTRAINT PK__Auth_use__610DFBB4519AEE00_2 PRIMARY KEY (auth_user_uid),
+	CONSTRAINT UQ_Auth_user_id_2 UNIQUE (user_id),
+);
+
+SET IDENTITY_INSERT Auth_user2 ON;
+
+INSERT INTO Auth_user2 (auth_user_uid,user_id,user_type,user_title,user_department,user_first_nm,user_last_nm,user_work_email,user_work_phone,
+	user_mobile_phone,master_sec_admin_ind,prog_area_admin_ind,nedss_entry_id,external_org_uid,user_password,user_comments,
+	add_time,add_user_id,last_chg_time,last_chg_user_id,record_status_cd,record_status_time,jurisdiction_derivation_ind,provider_uid)
+SELECT auth_user_uid,user_id,user_type,user_title,user_department,user_first_nm,user_last_nm,user_work_email,user_work_phone,
+	user_mobile_phone,master_sec_admin_ind,prog_area_admin_ind,nedss_entry_id,external_org_uid,user_password,user_comments,
+	add_time,add_user_id,last_chg_time,last_chg_user_id,record_status_cd,record_status_time,jurisdiction_derivation_ind,provider_uid FROM Auth_user where user_id in ('state', 'superuser', 'coordinator', 'clerical');
+
+ CREATE NONCLUSTERED INDEX IDX_Auth_user_02162021_04_2 ON dbo.Auth_user2 (  nedss_entry_id ASC  )  
+	 INCLUDE ( user_first_nm , user_last_nm ) 
+	 WITH (  PAD_INDEX = OFF ,FILLFACTOR = 100  ,SORT_IN_TEMPDB = OFF , IGNORE_DUP_KEY = OFF , STATISTICS_NORECOMPUTE = OFF , ONLINE = OFF , ALLOW_ROW_LOCKS = ON , ALLOW_PAGE_LOCKS = ON  )
+	 ON [PRIMARY ] ;
+
+CREATE TABLE NBS_ODSE.dbo.Entity_loc_participation_hist2 (
+	entity_uid bigint NOT NULL,
+	locator_uid bigint NOT NULL,
+	version_ctrl_nbr smallint NOT NULL,
+	add_reason_cd varchar(20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	add_time datetime NULL,
+	add_user_id bigint NULL,
+	cd varchar(50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	cd_desc_txt varchar(100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	class_cd varchar(10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	duration_amt varchar(20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	duration_unit_cd varchar(20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	from_time datetime NULL,
+	last_chg_reason_cd varchar(20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	last_chg_time datetime NULL,
+	last_chg_user_id bigint NULL,
+	locator_desc_txt varchar(2000) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	record_status_cd varchar(20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	record_status_time datetime NULL,
+	status_cd char(1) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+	status_time datetime NOT NULL,
+	to_time datetime NULL,
+	use_cd varchar(20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	user_affiliation_txt varchar(20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	valid_time_txt varchar(100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+	as_of_date datetime NULL,
+	CONSTRAINT PK_Entity_loc_particip_hist2 PRIMARY KEY (entity_uid,locator_uid,version_ctrl_nbr)
+);
+
+INSERT INTO Entity_loc_participation_hist2 
```

Comment:
question: Does it make sense to name the `Person2` something like `Person_shrunken` and `Person3` table as `Person_original`?

Reply:
only reason i kept it short is because the suffix is used for the new constraint/indexes and key names and they have a size limit but we should be ok